src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -53,13 +54,14 @@ export const flagNewSubmission = submissionId => {
   sessionStorage.setItem('new-submission', submissionId)
 }
 
+const collapsedStatuses = workflowStatuses => _.flow(
+  _.toPairs,
+  _.map(([status, count]) => ({ [collapseStatus(status)]: count })),
+  _.reduce(_.mergeWith(_.add), {})
+)(workflowStatuses)
```

Comment:
tiny nit: you could take advantage of currying and eliminate the explicit `workflowStatuses` parameter, if you wanted.